project-example/week-7/weather-app-without-searchbar/server/.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
```

Comment:
Is this right? Maybe it should match the one you have in client:

```
.env.local
.env.development.local
.env.test.local
.env.production.local
```

because you have the `.env` file checked in above.

Reply:
I had accidentally committed .env file earlier. Later deleted my the API key & created .gitignore. Thats why it is here. Thank you for reviewing the project.